### ISSUE TITLE
[1.10.2] [Bug] Crash to desktop upon entering the nether

### ISSUE THREAD

**Issue by SnowShock35** [progwml6/Natura#271]
Hello,

I believe to have found a bug with Natura for Minecraft 1.10.2, here are the details. . . 
### Important Information
- Minecraft 1.10.2 
- Minecraft Forge 12.18.1.2094
- Mantle 1.0.0
- Natura 1.10.2-4.0.0.72
### Issue Description

When entering the nether the world crashes directly to desktop. This will continue to happen every time you re-open the world. I believe it's perhaps the cause of Bloodwood saplings not being planted on the right side of the block but I'm not 100% certain. 
### How to Reproduce
1. Load up an instance with the above mods/dependancies
2. Create a world
3. Make a nether portal, ignite it and then go through
4. Crash to desktop 
### Issue Media

[Crash Report via FTB Paste](http://paste.feed-the-beast.com/view/0fc66e03)

**Comment by owmii** [comment]
same issue :P 

```
Description: Exception ticking world

java.lang.ClassCastException: net.minecraft.block.state.BlockStateContainer$StateImplementation cannot be cast to net.minecraftforge.common.IPlantable
    at com.progwml6.natura.world.worldgen.berry.nether.NetherBerryBushGenerator.findGround(NetherBerryBushGenerator.java:210)
    at com.progwml6.natura.world.worldgen.berry.nether.NetherBerryBushGenerator.generateBush(NetherBerryBushGenerator.java:30)
    at com.progwml6.natura.world.worldgen.BerryBushGenerator.generateNether(BerryBushGenerator.java:175)
    at com.progwml6.natura.world.worldgen.BerryBushGenerator.generate(BerryBushGenerator.java:53)
    at net.minecraftforge.fml.common.registry.GameRegistry.generateWorld(GameRegistry.java:120)
    at net.minecraft.world.chunk.Chunk.func_186034_a(Chunk.java:1001)
    at net.minecraft.world.chunk.Chunk.func_186030_a(Chunk.java:974)
    at net.minecraft.world.gen.ChunkProviderServer.func_186025_d(ChunkProviderServer.java:154)
    at net.minecraft.world.World.func_72964_e(World.java:303)
    at net.minecraft.world.World.func_175726_f(World.java:298)
    at net.minecraft.world.World.func_180495_p(World.java:842)
    at net.minecraft.world.chunk.Chunk.func_76594_o(Chunk.java:1257)
    at net.minecraft.world.WorldServer.func_147456_g(WorldServer.java:393)
    at net.minecraft.world.WorldServer.func_72835_b(WorldServer.java:220)
    at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:698)
    at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613)
    at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:240)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471)
    at java.lang.Thread.run(Thread.java:745)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
    at com.progwml6.natura.world.worldgen.berry.nether.NetherBerryBushGenerator.findGround(NetherBerryBushGenerator.java:210)
    at com.progwml6.natura.world.worldgen.berry.nether.NetherBerryBushGenerator.generateBush(NetherBerryBushGenerator.java:30)
    at com.progwml6.natura.world.worldgen.BerryBushGenerator.generateNether(BerryBushGenerator.java:175)
    at com.progwml6.natura.world.worldgen.BerryBushGenerator.generate(BerryBushGenerator.java:53)
```

**Comment by alexbegt** [comment]
Fixed in ba8ef0058ca80e2c8ebe228d40332038ffec4cea.
